### PR TITLE
feature/#80 improve datanode filtering documentation

### PR DESCRIPTION
**Pull request by toan-quach**
#80

### CODE REVIEW

File: docs/manuals/core/entities/data-node-mgt.md [hunk 1]
```diff
@@ -99,12 +99,16 @@ stored on the data node according to the type of data node:
 
 It is also possible to partially read the contents of data nodes, which comes in handy when dealing with large amounts
 of data.
-This can be achieved using the `DataNode.filter()^` method:
+This can be achieved by providing an operator, a Tuple of (key, value, comparison operator),
+or a list of operators to the `DataNode.filter()^` method:
 
 ```python linenums="1"
 data_node.filter([('field_name_like_temperature', 14, Operator.EQUAL), ('field_name_like_temperature', 10, Operator.EQUAL)], JoinOperator.OR))
```
Comment (jrobinAV) [inline docs/manuals/core/entities/data-node-mgt.md]:
```suggestion
data_node.filter([('field_name', 14, Operator.EQUAL), ('field_name', 10, Operator.EQUAL)], JoinOperator.OR))
```

**Review by jrobinAV — APPROVED** [review]

File: docs/manuals/core/entities/data-node-mgt.md [hunk 2]
```diff
@@ -99,16 +99,20 @@ stored on the data node according to the type of data node:
 
 It is also possible to partially read the contents of data nodes, which comes in handy when dealing with large amounts
 of data.
-This can be achieved using the `DataNode.filter()^` method:
+This can be achieved by providing an operator, a Tuple of (field_name, value, comparison operator),
+or a list of operators to the `DataNode.filter()^` method:
 
 ```python linenums="1"
-data_node.filter([('field_name_like_temperature', 14, Operator.EQUAL), ('field_name_like_temperature', 10, Operator.EQUAL)], JoinOperator.OR))
+data_node.filter([('field_name', 14, Operator.EQUAL), ('field_name', 10, Operator.EQUAL)], JoinOperator.OR))
```
Comment (FabienLelaquais) [inline docs/manuals/core/entities/data-node-mgt.md]: Double-quotes should be preferred over single-quotes.
  You are more likely to have strings (delimited by double-quotes) containing single-quote character than the other way aroud.

File: docs/manuals/core/entities/data-node-mgt.md [hunk 3]
```diff
@@ -99,16 +99,20 @@ stored on the data node according to the type of data node:
...
 ```
 
+If a list of operators is provided, it is necessary to provide a join operator that will be
+used to combine the filtered results from the operators.
+
 It is also possible to use pandas style filtering:
 
 ```python linenums="1"
-temp_data = data_node['field_name_like_temperature']
+temp_data = data_node['field_name']
```
Comment (FabienLelaquais) [inline docs/manuals/core/entities/data-node-mgt.md]: Double-quotes should be preferred over single-quotes.

File: docs/manuals/core/entities/data-node-mgt.md [hunk 4]
```diff
@@ -99,16 +99,20 @@ stored on the data node according to the type of data node:
 
 It is also possible to partially read the contents of data nodes, which comes in handy when dealing with large amounts
 of data.
-This can be achieved using the `DataNode.filter()^` method:
+This can be achieved by providing an operator, a Tuple of (field_name, value, comparison operator),
```
Comment (FabienLelaquais) [inline docs/manuals/core/entities/data-node-mgt.md]: I suspect that [field_name, value, comparison operator]  should be in italics...

**Review by FabienLelaquais — APPROVED** [review]
Looks good!

**Pull request merged**